arduino-ide-extension/src/browser/dialogs/ide-updater/ide-updater-dialog.tsx

```diff
@@ -76,11 +74,15 @@ export class IDEUpdaterDialog extends ReactDialog<UpdateInfo | undefined> {
         <IDEUpdaterComponent
           updateInfo={this.updateInfo}
           updateProgress={this.updateProgress}
+          openExternal={this.openExternal}
         />
       )
     );
   }
 
+  private readonly openExternal = (url: string) =>
+    this.windowService.openNewWindow(url, { external: true });
```

Comment:
> @kittaakos what about adding validation on the URL parameter to accept only `https` and `http` URLs?
> It helps to follow a security-by-design strategy in case of code evolution

I appreciate your input. This is a great idea but unrelated to this PR and the Theia version update. I plan to keep this the same in the context of this PR. Thanks for your understanding!

----

I remember bringing up this topic in one of the internal chats, please open a follow-up either in [arduino/arduino-ide](https://github.com/arduino/arduino-ide/) or directly to [eclipse-theia/theia/](https://github.com/eclipse-theia/theia).

There are a few related issues in Theia:
 - [vscode: Support trusted commands in markdown and webviews](https://github.com/eclipse-theia/theia/issues/12079)
 - [[vscode] Implement full Workspace Trust behavior](https://github.com/eclipse-theia/theia/issues/12318)

Here is the corresponding VS Code documentation: https://code.visualstudio.com/docs/editor/workspace-trust
Here is how VS Code handles trusted domains: https://github.com/microsoft/vscode/blob/a54b497150197f773f485038f44af02772c5a016/src/vs/workbench/contrib/url/browser/trustedDomains.ts

Reply:
> adding validation on the URL

For completeness, this is the expected place for having such URL validation in IDE2:

https://github.com/arduino/arduino-ide/blob/964ea3bc0cb51c87592757ca7b8b73002c475033/arduino-ide-extension/src/electron-main/theia/electron-main-window-service.ts#L24-L26

Reply:
This topic is not tracked at https://github.com/arduino/arduino-ide/issues/2056.

I am resolving this thread.